src/app/contextBroker/contextBroker.cpp

```diff
@@ -1481,14 +1482,22 @@ int main(int argC, char* argV[])
   paConfig("man description",               (void*) description);
   paConfig("man author",                    (void*) "Telefonica I+D");
   paConfig("man version",                   (void*) ORION_VERSION);
-  paConfig("log to screen",                 (void*) true);
   paConfig("log to file",                   (void*) true);
   paConfig("log file line format",          (void*) LOG_FILE_LINE_FORMAT);
-  paConfig("screen line format",            (void*) "TYPE@TIME  FILE[LINE]: TEXT");
+  paConfig("log file time format",          (void*) "%Y-%m-%dT%H:%M:%S");
   paConfig("builtin prefix",                (void*) "ORION_");
   paConfig("usage and exit on any warning", (void*) true);
   paConfig("no preamble",                   NULL);
-  paConfig("log file time format",          (void*) "%Y-%m-%dT%H:%M:%S");
+
+
+  //
+  // If option '-fg' is set, print traces to stdout as well, otherwise, only to file
+  //
+  if (paIsSet(argC, argV, "-fg"))
```

Comment:
We should also include some comment about the new way of working in the .md doc. I can contribute to the PR with that (it is easy for me find the points to edit) but an English proof read will be welcome as usual :)